packages/storybook/.storybook/style.scss

```diff
@@ -4,3 +4,15 @@ $formation-asset-path: '~@department-of-veterans-affairs/formation/assets';
 @import '@department-of-veterans-affairs/formation/sass/base/_b-variables.scss';
 @import '@department-of-veterans-affairs/formation/sass/base/_b-breakpoints.scss';
 @import '@department-of-veterans-affairs/formation/sass/site/_m-crisis-line.scss';
+
+// Remove Negative Margin via Default Storybook CSS
+// This is causing the show code button to overlap with components
+// Height Auto fixes Firefox Bug with height being set to a fixed 70px value
+.docs-story {
+    & > div {
+        margin: 0;
+        & > div {
+            height: auto;
+        }
+    }
+}
```

Comment:
This is fantastic! the `height: auto` also makes the preview resize dynamically which it didn't do before:

![Peek 2021-11-30 11-58](https://user-images.githubusercontent.com/2008881/144118930-c340aa99-40fb-4b47-a30b-41b6a18480f4.gif)

Reply:
Woot!